wire-runtime/src/nativeMain/kotlin/com/squareup/wire/internal/-Platform.kt

```diff
@@ -19,6 +19,12 @@ import okio.IOException
 
 actual interface Serializable
 
+/** This annotation is an empty placeholder. */
+actual annotation class JvmField
+
+/** This annotation is an empty placeholder. */
+actual annotation class JvmStatic
```

Comment:
the problem is that `OptionalExpectation` can only be used in common source sets

Reply:
This is the same reason I could not use the one in common stdlib I think

Reply:
I thought that it might work since I tried _before_ I fixed the typealias resolution.
But 
`Declaration annotated with '@OptionalExpectation' can only be used in common module sources`

Reply:
I believe what Jake is saying is that we can just generate code with `kotlin.jvm.Jvm*` by default, since these types are declared in common and are themselves marked with `@OptionalExpectation`. Hence we don't need our own types, nor do we need the `jvmOnly` opt-in to use `kotlin.jvm.Jvm*` annotations in generated code.

Reply:
This is the same problem. Since the generated code is not in a common module source, it won't compile because of
`Declaration annotated with '@OptionalExpectation' can only be used in common module sources`